external/netstandard/netstandard.depproj

```diff
@@ -76,7 +76,19 @@
     </BinPlaceConfiguration>
   </ItemGroup>
 
-  <Target Name="AddNETStandardRefs" AfterTargets="ResolveReferences"
+  <Target Name="AddNETStandard21Refs" AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <_NetStandard21Files Include="$(PackagesDir)$(NETStandardLibraryPackageId.ToLower())\$(NETStandardLibraryPackageVersion)\build\netstandard2.1\ref\*.dll" />
+    </ItemGroup>
+
+    <Error Condition="'@(_NetStandard21Files)' == ''" Text="Could not find package assets for netstandard2.1" />
+
+    <Copy SourceFiles="@(_NetStandard21Files)"
```

Comment:
Should we be excluding S.CM.Composition, like the target below?
```
 <ExcludeNetStandardRefs Include="System.ComponentModel.Composition" />
```

Reply:
Possibly, looks like we started excluding it for 2.0 with Wes' change here: https://github.com/dotnet/corefx/commit/18b6befe0fc3ea12bd41f5d3a7e3461ef7fb1554. @weshaggard do you remember why we needed to exclude S.CM.Composition from binplacing for netstandard2.0?

Reply:
Yes because we started shipping Systeom.ComponentModel.Composition in its own package.